src/services/textChanges.ts

```diff
@@ -1361,7 +1361,10 @@ namespace ts.textChanges {
                     break;
 
                 default:
-                    if (isImportClause(node.parent) && node.parent.name === node) {
+                    if (node.parent === node) {

```

Comment:
Huh? AFAIK this can never be true. The crash presumably happens when `node.parent` is `undefined`, which means you’re still going to get the same crash on the next `else if`.

Reply:
Bah!  Sorry -- this was a debugging leftover that I should have removed.